src/main/preference.js

```diff
@@ -57,6 +58,29 @@ class Preference {
       })
     })
   }
+
+  update () {
+    if (this.requiresUpdate) {
+      this.requiresUpdate = false
+
+      // write latest settings to filesystem
+      const userSettings = this.getAll()
+      return this.setItem('theme', userSettings.theme)
```

Comment:
A little confused, only set the theme?

Reply:
Yes... `cache` has the latest settings combined with user settings and `this.setItem('theme', userSettings.theme)` triggers the save. It's not a perfect solution but no redundant code.